src/scenarios/shared/precommands.py

```diff
@@ -50,16 +52,27 @@ def __init__(self):
         args = parser.parse_args()
         self.configuration = args.configuration
```

Comment:
It breaks here when operation is RESTORE because it doesnt have configuration as arg

Reply:
Good catch! As discussed I'm just going to remove the `restore` verb.